src/index.js

```diff
@@ -0,0 +1,26 @@
+import './style.css';
+import addNew from './modules/postContent.js';
+import showgameData from './modules/getContent.js';
+
+const addBtn = document.querySelector('.submitbutton');
+const emptyContainer = document.querySelector('.emptycontent');
+const addName = document.querySelector('.addname');
+const addScore = document.querySelector('.addscore');
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (!addName.value || !addScore.value) {
+    emptyContainer.innerHTML = '<p class="alert">Please add name and score</p>';
+  } else {
+    addNew();
+  }
+});
+
+// show data
+showgameData();
+
+// Show data
+const refreshBtn = document.querySelector('.refresh');
+refreshBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  window.location.reload();
+});
```

Comment:
- [ ] Here, I noticed that when the Refresh button is clicked, you then reload the entire page in order to display the new scores data. However, the project requirement states that when the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change. Please implement this requirement 👍

Reply:
Added required  changes
Open the video it shows how it works: https://www.loom.com/share/d92b94e1d20c47488ffcc05869e65f93